### PR TITLE
fixed #141

### LINKED ISSUE #141: ignores branch/tag argument from cmd
The call to `highest()` at https://github.com/adaptlearning/adapt-cli/blob/d68d1e7083a3daeb5929f7ee81faf4b0a9e0218a/lib/commands/create.js#L18-L25 ignores the user's input of branch/tag stored in variable `branch` immediately before it.

To replicate:

1. Create a course with arguments as specified by README:
`adapt create course "My Course" legacy`  
2. The user is asked to confirm each argument provided except "leagacy". It is ignored and replaced by the highest tag.
cli-bug

### PR DESCRIPTION
**Pull request by NayanKhedkar**

### CODE REVIEW

File: lib/commands/create.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = {
                 return confirm({
                     type: type,
                     localDir: localDir,
-                    branch: tag,
+                    branch: branch || tag,
```
Comment (tomgreenfield) [inline lib/commands/create.js:23]: `branch || tag` defaults to `master` instead of the highest tag.

**Review by tomgreenfield — CHANGES_REQUESTED** [review]
This breaks https://github.com/adaptlearning/adapt-cli/pull/131.

File: lib/commands/create.js (hunk 1 above)
Comment (tomgreenfield) [reply lib/commands/create.js:23]: Should `DefaultBranch` now be removed from the constants?

Comment (NayanKhedkar) [reply lib/commands/create.js:23]: granted

**Review by tomgreenfield — APPROVED** [review]

**Review by moloko — APPROVED** [review]
code 👓

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**